jerry-core/lit/lit-char-helpers.cpp

```diff
@@ -328,3 +328,73 @@ lit_char_is_word_char (ecma_char_t c) /**< code unit */
           || (c >= LIT_CHAR_ASCII_DIGITS_BEGIN && c <= LIT_CHAR_ASCII_DIGITS_END)
           || c == LIT_CHAR_UNDERSCORE);
 } /* lit_char_is_word_char */
+
+/**
+ * Returns the lowercase character sequence of an ecma character.
+ *
+ * Note: output_buffer_p must be able to hold at least LIT_MAXIMUM_OTHER_CASE_LENGTH characters.
```

Comment:
@zherczeg, could you, please, add arguments like `buffer_size` to the function, and check the condition with `JERRY_ASSERT`?